content/cloud-computing/cloud-computing.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Cloud Computing'
+Description: 'Cloud computing refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cloud-computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Cloud computing** refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user. This is becoming popular with businesses and ubiquitous in personal computing. For example, whenever a person sets up a website on GoDaddy, plays World of Warcraft, or streams a movie on Netflix, they are using a cloud computing solution.
```

Comment:
> **Cloud computing** refers to an architecture where computing resources

I like this approach of bolding a topic (in addition to a concept or term) 🚀 Let's do this for all new topics going forward! 😄

Reply:
I made a similar suggestion down in the `go.md` file.